composer.json

```diff
@@ -28,12 +28,12 @@
         "psr/http-message": "^1.1 || ^2.0",
         "slim/slim": "^4.12",
         "symfony/polyfill-php81": "^1.29",
-        "twig/twig": "^3.8"
+        "twig/twig": "^3.11.1"
     },
     "require-dev": {
         "phpspec/prophecy-phpunit": "^2.0",
         "phpstan/phpstan": "^1.10.59",
-        "phpunit/phpunit": "^9.6 || ^10",
```

Comment:
Is this change needed?

Reply:
Not 100% sure, let me check and I'll update my branch or leave a comment here.

Reply:
This change has been reverted, it was due to my tests.